src/cli/git-proxy-cli.js

```diff
@@ -0,0 +1,148 @@
+const axios = require('axios');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const fs = require('fs');
+
+// Git-Proxy UI URL (configurable via environment variable)
+const { GIT_PROXY_UI_URL = 'http://localhost:8080' } = process.env;
```

Comment:
Can we pull the variable from existing functionality instead?

https://github.com/finos/git-proxy/blob/main/src/config/env.js

Reply:
I can reuse the GIT_PROXY_UI_PORT from env.js but will still need a variable to make the host configurable to support remote git-proxy management via this CLI tool. To go down that route, there would be the GIT_PROXY_UI_PORT and e.g. a GIT_PROXY_UI_HOST which two variables then would be added together to create the GIT_PROXY_UI_URL constant. 

Is this what you would like?

Reply:
done